introduction.md

```diff
@@ -0,0 +1,3 @@
+Computer science is a passion since I was very young. My interest in this area is focused on new technologies, development, GNU/Linux, open source software and all about the Google ecosystem.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.